code/game/objects/items/devices/instruments.dm

```diff
@@ -140,6 +140,13 @@
 	w_class = WEIGHT_CLASS_SMALL
 	actions_types = list(/datum/action/item_action/instrument)
 
+/obj/item/device/instrument/harmonica/proc/speechModification(message)

```

Comment:
Hack. Stop now.

Move the base ```speechModification``` to ```/obj/item```, rather than having an extra definition on this type.